core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/allExcept.kt

```diff
@@ -282,4 +311,79 @@ class AllExceptTests : ColumnsSelectionDslTests() {
             },
         ).shouldAllBeEqual()
     }
+
+    @Test
+    fun `except on column group`() {
+        val firstNameAccessor = column<String>("firstName")
+        listOf(
+            df.select { name }.remove { name.firstName }.alsoDebug(),
+            df.select { cols(name) except name.firstName },
+            df.select { name.except { cols { "first" in it.name } } },
+            df.select { name.except { cols { "first" in it.name } and cols { "first" in it.name } } },
+            df.select { name.except { firstName } },
+            df.select { name.except { firstNameAccessor } },
+            df.select { name.except { firstName and firstName } },
+            df.select { name.except { firstNameAccessor and firstNameAccessor } },
+//            df.select { name.except(name.firstName and name.firstName) }, // not allowed
```

Comment:
Looks like we need a test for this, but it's not possible in that manner and should be kind of tests for compiler plugin, when it's a String which should be compiled or not

Reply:
Or could we make assert on that?

Reply:
yes indeed, that's a good idea for later :)